File: src/main/java/no/nav/personopplysninger/features/kodeverk/api/KodeOgTekstDto.java
package no.nav.personopplysninger.features.kodeverk.api;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class KodeOgTekstDto {
    private String kode;
    private String tekst;

    public KodeOgTekstDto(String kode, String tekst) {
        this.kode = kode;
        this.tekst= tekst;
    }

    public String getKode() {
        return kode;
    }

    public String getTekst() {
        return tekst;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/oppslag/kodeverk/api/KodeOgTekstDto.kt
package no.nav.personopplysninger.features.oppslag.kodeverk.api

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class KodeOgTekstDto(
        @JsonProperty("kode") val kode: String,
        @JsonProperty("tekst") val tekst: String
)
